spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -1562,7 +1562,7 @@ def find_after_step_hooks(test_case)
               end
             end
             expect( formatter.legacy_messages ).to eq([
-                :before_features,
+                                                        :before_features,
                   :before_feature,
```

Comment:
Ooops I haven't notice the conflict, sorry for that. It would be good to synchronise to avoid repeated work /cc @pmatsinopoulos .